antora-playbook.yml

```
@@ -56,7 +56,7 @@ output:
 runtime:
   log:
     level: info
-    failure_level: warn
+    failure_level: error
```

```suggestion
    failure_level: warn
```

Keep failure at warning level.

---

This is important to catch alerts related to undeclared attributes (the current situation).